src/parquet/CMakeLists.txt

```diff
@@ -50,6 +52,7 @@ install(FILES
   "${CMAKE_CURRENT_BINARY_DIR}/parquet.pc"
   DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig/")
 
+ADD_PARQUET_TEST(bloom-test)
```

Comment:
Is this only used in the test? What must be done to actually integrate it with the rest of the repository and get it into some binary tools or libraries that can be used by consumers of parquet-cpp build artifacts?

Reply:
Yes, this is only expected to be used in test. I don't expect this unit test to be contained in final binary file. I think this test is used to verify correctness and compatibility before parquet-cpp releasing.

Reply:
I don't think I understand. If this file isn't used anywhere else in the project, how can it be actually used? @rdblue , maybe you could shed some light on how new features are usually integrated into the non-test parts of this project.

Reply:
this file is used when execute "make unittest" command, it run several unit test. But the unit test code is not linked into finally parquet-cpp binary.

Reply:
My bad, I thought it is not linked to, while I just see it is linked into final binary in compiling log.  Let me investigate how other unit test achieve this.

Reply:
Just reviewed this again, bloom-test.cc is not linked to either libparquet.a or libparquet.so, the logs I mentioned above is linking bloom-test.o to bloom-test binary in debug/bloom-test.

Reply:
I am not comfortable that this patch is complete until it is clearly used somewhere *other* than a test. If BFs are part of the format, and this repo includes utilities for reading or writing the format, they should include this code in their compiled binaries.

Reply:
I understand your concern while we should discussed in previous parquet sync-up meeting and agreed to implement BF utility in the test scope firstly since BFs are optional of metadata.  BFs are planed to be used when either read or write side is implemented, we should include some usages right in next steps, it that OK?

Reply:
OK. Please note as such in your commit message.